plugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -26,15 +27,19 @@
         <argument value="${writeTimeoutInMs}" />
         <argument value="${readTimeoutInMs}" />
         <argument value="${metricCacheSize}" />
+        <argument value="${externalCacheSize}" />
         <argument value="${bulkheadMaxWaitDuration}" />
         <argument value="${organizationId}" />
     </bean>
 
+    <!--Key-value store -->
+    <reference id="keyValueStore" interface="org.opennms.integration.api.v1.distributed.KeyValueStore" availability="optional"/>
+
     <bean id="timeSeriesStorage" class="org.opennms.timeseries.cortex.CortexTSS" destroy-method="destroy">
         <argument ref="cortexTssConfig" />
+        <argument ref="keyValueStore" />
     </bean>
 
-    <service ref="timeSeriesStorage" interface="org.opennms.timeseries.cortex.CortexTSS" />
```

Comment:
@j-white deleting this line doesn't let the shell command be registered in Karaf. Not sure if this is the reason of that line or there is a proper way to add shell commnds to the registry.

Reply:
That makes sense - we should put this one back then